firefox-ios/RustFxA/FxAWebViewModel.swift

```diff
@@ -243,50 +244,56 @@ extension FxAWebViewModel {
 
     // Handle a message coming from the content server.
     private func handleRemote(command rawValue: String, id: Int?, data: Any?, webView: WKWebView) {
-        if let command = RemoteCommand(rawValue: rawValue) {
-            switch command {
-            case .oauthLogin:
-                if let data = data {
-                    onLoginComplete(data: data, webView: webView)
-                } else {
-                    onDismissController?()
-                }
-            case .changePassword:
-                if let data = data {
-                    onPasswordChange(data: data, webView: webView)
-                }
-            case .status:
-                if let id = id {
-                    onSessionStatus(id: id, webView: webView)
-                }
-            case .deleteAccount, .signOut:
-                profile.removeAccount()
+        logger.log("webchannel message: \(rawValue)", level: .info, category: .sync)
+        let command = RemoteCommand(rawValue: rawValue) ?? .unknown
+        switch command {
```

Comment:
These are two lines where the actual logic exists: I separated the `if let` into two lines so that we have a default `.unknown` which lets us respond to those messages.

The diff looks complicated because of the un-indentation. 😕